Tails specific: we only want hdparm so that laptop-mode-tools can use it.

--- chroot.orig/etc/init.d/hdparm	2012-09-24 10:05:12.877043486 +0200
+++ chroot/etc/init.d/hdparm	2012-09-24 10:03:30.638108333 +0200
@@ -8,2 +8,2 @@
-# Default-Start:     S
+# Default-Start:
-# Default-Stop:
+# Default-Stop:      S
